CMakeLists.txt

```diff
@@ -5,11 +5,16 @@ project(projectname)
 list(APPEND CMAKE_MODULE_PATH "${PROJECT_SOURCE_DIR}/lib/cmake-modules")
 
 if(UNIX)
-	set(CMAKE_CXX_FLAGS "-m32 -fvisibility=hidden")
+	set(CMAKE_CXX_FLAGS "-m32 -fvisibility=hidden -std=c++11")
 	set(CMAKE_C_FLAGS "-m32 -fvisibility=hidden")
 	set_property(GLOBAL PROPERTY FIND_LIBRARY_USE_LIB64_PATHS OFF)
 endif()
 
+if(WIN32)
+	SET( CMAKE_LIBRARY_OUTPUT_DIRECTORY_DEBUG "${CMAKE_SOURCE_DIR}/test/plugins")
```

Comment:
This should use the same convention as the above `set` - lower case and no space preceding the first argument.